gradle/wrapper/gradle-wrapper.properties

```
@@ -1,6 +1,6 @@
 #Wed Feb 02 12:03:18 GMT 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.2-bin.zip
```

Gradle recommends using 8.0.2 instead of 8.0. From their release notes:

> We recommend users upgrade to 8.0.2 instead of 8.0.

---

Done!